testsuite/tests/typing-layouts-unboxed-records/unique.ml

```diff
@@ -0,0 +1,92 @@
+(* TEST
+ flambda2;
+ include stdlib_upstream_compatible;
+ flags = "-extension layouts_alpha -extension unique";
+ {
+   expect;
+ }
+*)
+
+(* Uniqueness tests *)
+
+type t = #{ x : string ; y : string }
+let use_string (_s @ unique) = ()
```

Comment:
This function could be `unique_use` since it accepts non-strings as well. That also avoids confusion later when we may or may not make strings mode-cross uniqueness

Reply:
Thanks! Updated.